openxla_patches/gpu_hanging.diff

```diff
@@ -0,0 +1,34 @@
+diff --git a/xla/service/gpu/gpu_executable.cc b/xla/service/gpu/gpu_executable.cc
```

Comment:
Can you add a comment with the correspond git commit hash in openXLA?

Reply:
done